tests/test_smoketest.py

```diff
@@ -52,13 +54,23 @@ def run_for_json(command, **kwargs):
 
 @testmode.integration
 class Smoketest(unittest.TestCase):
-    replicas = "aws", "gcp"
+    params = [
```

Comment:
I would do this instead:

```
REPLICAS = {
  Replica.AWS: {
    'checkout_bucket_envvar': "DSS_S3_CHECKOUT_BUCKET",
    'test_bucket_envvar': "DSS_S3_BUCKET_TEST",
  },
  Replica.GCP: {
    'checkout_bucket_envvar': "DSS_GS_CHECKOUT_BUCKET",
    'test_bucket_envvar': "DSS_GS_BUCKET_TEST",
  },
}
```

Reply:
Changes it to use Replica, but I kept it as a list.